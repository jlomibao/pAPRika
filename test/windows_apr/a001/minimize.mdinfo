

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2590  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5068  EEL     =     1332.3152  EGB        =     -127.4338
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7310  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================
