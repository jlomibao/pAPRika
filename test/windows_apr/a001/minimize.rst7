defa
  125
  -0.5269939   4.9601742   1.8322213   1.0094827   4.8954520   1.8332217
  -0.8876475   4.2860936   0.5923001  -0.8936111   4.2472842   3.0483187
  -0.9054754   5.9887518   1.8470422   1.4731936   5.8885048   1.8492845
   1.3141135   4.1940094   0.5936001   1.3134258   4.1534718   3.0491334
   2.6822569   4.0771320   0.1102719   2.6775804   4.0830955   3.5591965
   0.1936803   3.8252182   3.7686003   0.1990421   3.8904200  -0.1408692
   0.1726766   3.3266233   4.8805796   0.1806919   3.4242017  -1.2668002
   2.6674597   4.0322924  -0.9848131   3.2469145   4.9561723   0.4417719
   2.6423983   4.0513085   4.6542731   3.2221561   4.9750620   3.2286326
  -4.6463222   1.9698072   1.8306072  -3.8133988   3.2630861   1.8310247
  -4.2061508   1.2880466   0.6208567  -4.2598529   1.3245695   3.0783843
  -5.7248671   2.1634789   1.8046903  -4.4361702   4.1647854   1.8079163
  -3.0143821   3.1454487   0.6189372  -3.0629456   3.1764567   3.0777041
  -2.2603466   4.2820564   0.1076283  -2.2590964   4.2929957   3.5576890
  -3.3832194   2.0708330   3.8199683  -3.2860223   2.0078592  -0.0950420
  -3.0136393   1.8325092   4.9568747  -2.8512913   1.7283215  -1.1988747
  -2.2482430   4.2336221  -0.9873367  -2.7503763   5.2056804   0.4366144
  -2.2272202   4.2594692   4.6528150  -2.7262802   5.2273560   3.2258011
  -4.0858882  -3.0923302   1.8431195  -4.8051559  -1.7326141   1.8371118
  -3.3227493  -3.0716471   0.6016574  -3.2858423  -3.0406179   3.0593165
  -4.7829901  -3.9379414   1.8638614  -5.8965452  -1.8330226   1.8521885
  -4.3518099  -1.1222670   0.5942719  -4.3191740  -1.0905105   3.0510668
  -4.9247273   0.1267239   0.1124721  -4.9406397   0.1281210   3.5541059
  -3.4658109  -1.8840209   3.7718994  -3.5328188  -1.9390224  -0.1388120
  -3.0144729  -1.6394616   4.8773768  -3.1258855  -1.7279254  -1.2683232
  -4.8796549   0.1362589  -0.9826634  -5.9674612   0.1848075   0.4452958
  -4.9059384   0.1143873   4.6495730  -5.9816852   0.1586632   3.2124467
   0.5811078  -5.1295960   1.8492203  -0.9553810  -5.0642224   1.8501919
   0.9452613  -4.4236122   0.6284440   0.9444409  -4.4453207   3.0828147
   0.9600655  -6.1580667   1.8404877  -1.4202508  -6.0568341   1.8427679
  -1.2600731  -4.3305702   0.6294891  -1.2576289  -4.3508007   3.0838340
  -2.6244747  -4.2568186   0.1226423  -2.6246500  -4.2335082   3.5713574
  -0.1407780  -4.0380002   3.8119631  -0.1418041  -3.9993349  -0.0901307
  -0.1200099  -3.5554981   4.9309591  -0.1215834  -3.4941575  -1.1992176
  -2.5921521  -4.2289052  -0.9726281  -3.1722071  -5.1449585   0.4582553
  -2.6059305  -4.1841513   4.6661883  -3.1890093  -5.1149374   3.2457491
   4.6994460  -2.1363369   1.8356274   3.8675074  -3.4301476   1.8406356
   4.2974341  -1.4874637   0.5947231   4.2707978  -1.4572097   3.0510252
   5.7784557  -2.3286867   1.8490196   4.4902911  -4.3319578   1.8584840
   3.1062185  -3.3421722   0.6010067   3.0765878  -3.3128772   3.0582200
   2.3106480  -4.4637348   0.1205668   2.3168570  -4.4459650   3.5691417
   3.3536814  -2.1762041   3.7716120   3.4090997  -2.2292698  -0.1372600
   2.9240894  -1.8960156   4.8773077   3.0169733  -1.9817325  -1.2645953
   2.2801068  -4.4312873  -0.9746298   2.7821666  -5.3952827   0.4542256
   2.3034644  -4.3970384   4.6640699   2.8038137  -5.3716733   3.2414575
   4.1425895   2.9269159   1.8334706   4.8592784   1.5656614   1.8320831
   3.3393078   2.8809625   0.6193866   3.3852657   2.9033854   3.0788274
   4.8417484   3.7708399   1.8132559   5.9506876   1.6640208   1.8077746
   4.3623768   0.9262714   0.6210144   4.4156701   0.9541914   3.0778831
   4.9764992  -0.2932402   0.1121733   4.9924813  -0.2954057   3.5543952
   3.6032494   1.7711921   3.8180579   3.5079837   1.7235520  -0.0942127
   3.2084826   1.5609926   4.9520213   3.0488680   1.4814967  -1.1970375
   4.9314662  -0.2799926  -0.9829252   6.0206485  -0.3261341   0.4440010
   4.9558775  -0.3058799   4.6498364   6.0326771  -0.3530885   3.2136535
   0.0106037  -0.0709727  -0.0589403  -0.9035331  -0.6562623  -0.2169638
   0.8729006  -0.7297478  -0.2190828   0.0426292   0.7244865  -0.8135267
   0.0366980   0.5226306   1.3609518   0.9449030   1.1245418   1.4877045
  -0.0102130  -0.5803312   2.4378040  -0.8166923   1.1998100   1.4887932
  -0.9190476  -1.1812575   2.3108947   0.8426503  -1.2581091   2.3095974
   0.0168328   0.0112234   3.8583005  -0.8427003   0.6736735   4.0181505
  -0.0195001  -0.7852322   4.6116437   0.9331428   0.5926185   4.0181295
  -0.0000540   0.0004286  -6.0004087  -0.0000941  -0.0006315  -9.0000682
   0.0000573   2.2000034 -11.2000229
