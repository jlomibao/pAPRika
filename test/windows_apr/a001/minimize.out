
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 17:47:35

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/a001
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   915812 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -8.1356E+02     1.0057E+01     2.4770E+01     N12        57

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -67.7110  EEL     =     1326.2731  EGB        =     -123.8292
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -813.5649
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.2743E+02     5.0062E-01     1.6872E+00     H7        120

 BOND    =        1.2586  ANGLE   =      107.5584  DIHED      =       52.5860
 VDWAALS =      -68.4302  EEL     =     1332.0634  EGB        =     -127.3961
 1-4 VDW =        5.4796  1-4 EEL =    -2130.5468  RESTRAINT  =        0.0003
 EAMBER  =     -827.4270
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.2744E+02     4.9442E-01     1.5809E+00     H32       105

 BOND    =        1.2629  ANGLE   =      107.5464  DIHED      =       52.6098
 VDWAALS =      -68.5065  EEL     =     1332.3206  EGB        =     -127.4413
 1-4 VDW =        5.5022  1-4 EEL =    -2130.7317  RESTRAINT  =        0.0009
 EAMBER  =     -827.4376
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2589  ANGLE   =      107.5469  DIHED      =       52.6098
 VDWAALS =      -68.5067  EEL     =     1332.3148  EGB        =     -127.4337
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7308  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2590  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5067  EEL     =     1332.3149  EGB        =     -127.4336
 1-4 VDW =        5.5029  1-4 EEL =    -2130.7309  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2589  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5067  EEL     =     1332.3149  EGB        =     -127.4337
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7308  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2590  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5068  EEL     =     1332.3151  EGB        =     -127.4337
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7310  RESTRAINT  =        0.0009
 EAMBER  =     -827.4379
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2589  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5068  EEL     =     1332.3149  EGB        =     -127.4337
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7308  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2589  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5068  EEL     =     1332.3151  EGB        =     -127.4338
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7308  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2590  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5068  EEL     =     1332.3152  EGB        =     -127.4338
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7310  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2590  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5068  EEL     =     1332.3152  EGB        =     -127.4338
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7310  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9389E-01     1.5809E+00     H32       105

 BOND    =        1.2590  ANGLE   =      107.5468  DIHED      =       52.6098
 VDWAALS =      -68.5068  EEL     =     1332.3152  EGB        =     -127.4338
 1-4 VDW =        5.5030  1-4 EEL =    -2130.7310  RESTRAINT  =        0.0009
 EAMBER  =     -827.4378
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.301    9.302    0.000    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.360  147.398    0.038    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.480   49.483    0.003    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   61.991   62.003    0.012    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.158 -146.178    0.021    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.640  -72.548    0.092    0.000 t
  DUM  DM1    3 --  C    BUT    2:    5.942    6.000    0.058    0.000 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.284  180.000    0.716    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  178.109  180.000    1.891    0.000 a
                                       Total distance penalty:      0.000
                                       Total angle    penalty:      0.001
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.465
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.33 (22.40% of Gen B)
|                Calc gb diag               4.14 (39.81% of Gen B)
|                Calc gb off-diag           3.92 (37.76% of Gen B)
|                Other                      0.00 ( 0.03% of Gen B)
|             Gen Born time             10.39 (100.0% of Nonbo)
|          Nonbond force             10.39 (90.69% of Force)
|          Bond/Angle/Dihedral        1.03 ( 8.99% of Force)
|          Other                      0.04 ( 0.31% of Force)
|       Force time                11.46 (100.0% of Runmd)
|    Runmd Time                11.46 (99.78% of Total)
|    Other                      0.02 ( 0.22% of Total)
| Total time                11.48 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 17:47:35.911  on 04/10/2019
|           Setup done at 17:47:35.917  on 04/10/2019
|           Run   done at 17:47:47.394  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
